docs/types/value-types.rst

```diff
@@ -83,8 +83,8 @@ for example ``uint256(0) - uint256(1) == 2**256 - 1``. You have to take these ov
 into account when designing safe smart contracts.
 
 The expression ``-x`` is equivalent to ``(T(0) - x)`` where
-``T`` is the type of ``x``. This means that ``-x`` will not be negative
-if the type of ``x`` is an unsigned integer type. Also, ``-x`` can be
+``T`` is the type of ``x``. It can only be applied to signed types.
+The value of ``-x`` can be
 positive if ``x`` is negative. There is another caveat also resulting
```

Comment:
> The value of ``-x`` can be positive if ``x`` is negative. 

I'm a bit confused. Why does this have to be specified here?

Reply:
For some people, this is not obvious.

Reply:
I stumbled across that myself as well earlier - it does feel a bit silly :-).